### PR TITLE
Some new tool icons

### PR DESCRIPTION
**Pull request by Arokha**
2 welders, multitool

dreamseeker_2020-02-21_00-04-10

dreamseeker_2020-02-21_00-11-09

CEV-Eris

### CODE REVIEW

**Comment by Heroman3003** [comment]
Those just look more cluttered. Plus, it will make big disparity in welder sprites.

**Comment by Ketrai** [comment]
These are of much better quality. +1 from me. No more blurry low contrast sprites. Just saying they look cluttered is insulting.

**Comment by Arokha** [comment]
I mirrored them L/R so they face the same direction as the current ones for more matchyness.

**Comment by Heroman3003** [comment]
Tbh, if the welder sprites are to be changed, it should be all station welders, as we have 5 different types of them with same 'base' but different details and coloration. Just changing some of them just makes it clash needlessly.

File: code/game/objects/items/weapons/tools/weldingtool.dm [hunk 1]
```diff
@@ -448,6 +450,7 @@
 /obj/item/weapon/weldingtool/experimental/hybrid
 	name = "strange welding tool"
 	desc = "An experimental welder capable of synthesizing its own fuel from spatial waveforms. It's like welding with a star!"
+	icon = 'icons/obj/tools.dmi' //VOREStation Edit
```
Comment (Novacat) [inline code/game/objects/items/weapons/tools/weldingtool.dm]: This is probably not needed since experimental already reverts it to tools.dmi

**Comment by SpadesNeil** [comment]
I don't care for the aesthetics of these.

**Pull request closed without merging**